person.rb

```diff
@@ -0,0 +1,22 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  # constructor
+  def initialize(id, age, name = 'unknown', parent_permision: true)
+    @id = id
```

Comment:
- [ ] [Optional] [Highly recommended] It's a good practice and professional to generate the `id` automatically like the suggestion below, not as a parameter as you did here.
```ruby
def initialize(age, name = "Unknown", parent_permission: true)
    @id = Random.rand(1..1000)
    @age = age
```